src/host/os_getversion.c

```diff
@@ -75,10 +75,14 @@ SYSTEM_INFO getsysteminfo()
 
 void getversion(struct OsVersionInfo* info)
 {
-	OSVERSIONINFOEX versionInfo = {0};
+	RTL_OSVERSIONINFOEXW versionInfo = {0};
 
-	versionInfo.dwOSVersionInfoSize = sizeof(OSVERSIONINFOEX);
-	GetVersionEx((OSVERSIONINFO*)&versionInfo);
+	versionInfo.dwOSVersionInfoSize = sizeof(RTL_OSVERSIONINFOEXW);
+
+	typedef LONG (WINAPI* RtlGetVersionSig)(PRTL_OSVERSIONINFOEXW);
+	RtlGetVersionSig getVersion = (RtlGetVersionSig)
+	GetProcAddress(GetModuleHandle(TEXT("ntdll.dll")), "RtlGetVersion");
+	if ( getVersion ) getVersion(&versionInfo);
```

Comment:
None of this should be used anymore...
"on par with the present "
would be using the recommended microsoft API's for user applications... "Rtl" methods are typically reserved for driver developement.

Instead on par would be using these new API's:
https://msdn.microsoft.com/en-us/library/windows/desktop/dn905474(v=vs.85).aspx
https://msdn.microsoft.com/en-us/library/windows/desktop/dn424964(v=vs.85).aspx

granted they feel somewhat weird... but that's what MS is recommending.